packages/easy-coding-standard/packages/SniffRunner/ValueObject/File.php

```diff
@@ -24,14 +25,14 @@
 final class File extends BaseFile
 {
     /**
-     * Explicit list for classes that use only warnings. ECS only knows only errors, so this one promotes them to error.
+     * Explicit list for classes that use only warnings. ECS only reports errors, so this one promotes them to error.
      *
      * @var array<class-string<Sniff>>
      */
-    private const REPORT_WARNINGS_SNIFFS = [
-        '\PHP_CodeSniffer\Standards\Generic\Sniffs\CodeAnalysis\AssignmentInConditionSniff',
+    private array $reportWarnings = [
```

Comment:
Why this change?

Reply:
Changed from constant to property because when set the option `Option::REPORT_WARNINGS` the classes are appended to this variable.

Is that ok?

Or you would prefer to maintain it as a constant and check for *report-warnings* listed in the constant and also check for (additional) *report-warnings* defined in `Option::REPORT_WARNINGS`?

Please advise.

Reply:
Resolved on commits 52c36f3 and 686ece2

Reply:
This should be only constant list of all sniffs that have warning only status.

Reply:
Ok, resolved at 55ba156

`REPORT_WARNINGS_SNIFFS` is a constant list (again).
`$reportWarnings` is a property that can be set with ~`appendReportWarnings`~`setReportWarnings`.